cocotb/share/lib/embed/gpi_embed.c

```diff
@@ -112,21 +112,10 @@ void embed_init_python(void)
 {
     FENTER;
 
-#ifndef PYTHON_SO_LIB
-#error "Python version needs passing in with -DPYTHON_SO_VERSION=libpython<ver>.so"
-#else
-#define PY_SO_LIB xstr(PYTHON_SO_LIB)
-#endif
-
     // Don't initialize Python if already running
     if (gtstate)
         return;
 
-    void * lib_handle = utils_dyn_open(PY_SO_LIB);
```

Comment:
Does `utils_dyn_open` have any callers remaining?

Reply:
Ah yes, it does - the gpi libraries.

Reply:
As I see it is used to load libraries specified in `GPI_EXTRA`.